### DISCUSSION TITLE
Combine all Security Updates into one PR?

### DISCUSSION THREAD

**Discussion by zjullion** [renovatebot/renovate#15553] [Request Help] [answered]
When using the GitHub app, is it possible to configure renovate to combine all security updates into a single PR? Is there some config that could be applied to `vulnerabilityAlerts` to make this happen?

**Comment by rarkins** [comment, marked as answer]
The existing code which removes grouping is here: https://github.com/renovatebot/renovate/blob/bc9eef228de3c9faa2331d856a17a3cc70dbc35b/lib/config/options/index.ts#L1512

You might be able to do this:

```
{
  "vulnerabilityAlerts": {
    "groupName": "Vulnerable dependencies"
  }
}
```

**Reply by zjullion** [reply]
That works! Thanks!

**Reply by hayawata3626** [reply]
@rarkins 
I would like to confirm one thing.
Is it possible to separate PRs by major version, minor version, and patch version as mentioned above?

**Reply by rarkins** [reply]
@hayawata3626 please start your own discussion and provide more details of what you mean. Your question is too broad, and I don't even know what you are referring to with "above" as there's multiple comments above and none seem applicable
